frontend/src/services/periodService.js

```diff
@@ -38,11 +38,11 @@ const getWeekInterval = (week) => {
     return moment.duration({w: week}).asSeconds();
 };
 
-export const getTimeByPeriod = (value) => {
+export const getTimeByPeriod = (period) => {
     let interval = null;
     let startDate = null;
     let endDate = null;
-    switch (value) {
+    switch (period) {
```

Comment:
If you don't changit it bake you break the logic period.PERIOD_TODAY

Reply:
I think changes in this file should be reverted entirely (see my last review)